app/controllers/accountants_controller.rb

```diff
@@ -1,9 +1,23 @@
 class AccountantsController < ApplicationController
   def index
-    head :ok
   end
 
   def search
-    head :ok
+    @results = Patient.search params[:search]
+
+    @patient = Patient.new
```

Comment:
These four instance variables are here to autopopulate some stuff in the call list builder -- don't think they're doing anything here. Same deal with the `searched_for` private methods below I think?